pkg/restAPI/requestListAvailableExtensions.go

```diff
@@ -24,6 +24,7 @@ func ListAvailableExtensions(apiContext *ApiContext) *openapi.Get {
 				Extensions: []ExtensionsResponseExtension{{
 					Id:                  "s3-vs",
 					Name:                "S3 Virtual Schema",
+					Category:            "virtual-schema",
```

Comment:
This file is evaluated. This part is just an example for a response that will be used in the generated API documentation. It will not be used in production.